vendor/guava-26_0-jre/build.gradle

```diff
@@ -29,6 +29,13 @@ vendorJava(
     "com.google.common": "org.apache.beam.vendor.guava.v26_0_jre.com.google.common",
     "com.google.thirdparty": "org.apache.beam.vendor.guava.v26_0_jre.com.google.thirdparty",
   ],
+  exclusions: [
```

Comment:
How are these exclusions relevant to the bugs listed, which seem to be about updating Guava (not its transitive dependencies)?

Reply:
To upgrade our vendored usage of guava we have to create an updated version of that artifact. We are currently using 20.0 and I checked in a previous PR that would allow building a vendored version of 26.0-jre but Kenn correctly pointed out that the artifact was incorrectly created due to it containing all these additional classes from transitive dependencies so I have added these exclusions to prevent these classes from appearing in the jar so I can work on producing the next release candidate.